### PR TITLE
[WIP] New theme

### PR DESCRIPTION
**Pull request by maciejwalkowiak**
Replaced current theme with https://github.com/kippt/jekyll-incorporated adjusted to our needs.

Few screenshots:
screen shot 2016-07-21 at 14 56 05
screen shot 2016-07-21 at 14 56 09
screen shot 2016-07-21 at 14 56 12
screen shot 2016-07-21 at 14 56 18

Todo: find better cover photo.

### CODE REVIEW

**Comment by florina-muntenescu** [comment]
the code block is not aligned with the text.

**Comment by florina-muntenescu** [comment]
the previous theme was automatically putting a new line after the picture description. is it possible to have the same here?

File: _includes/_google-analytics.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<!--google analytics tracking code here-->
```
Comment (mfernandezpajares) [inline _includes/_google-analytics.html:1]: this page is thinking on future features?

File: index.html [hunk 2]
```diff
@@ -1,12 +1,56 @@
...
+      <div class="meta">
+        Written By <address>{{ site.data.authors[post.author].name }}</address> &mdash;
+        <time pubdate datetime="{{ post.date | date: "%Y-%d-%B" }}" title="{{ post.date | date: "%B %d, %Y" }}">{{ post.date | date: "%B %d, %Y" }}</time>
+      </div>
+    </div>
+    <hr>
+  </section>
+  {% endunless %}
+  {% endfor %}
+  <!--<section class="pagination" style="text-align:center">
```
Comment (mfernandezpajares) [inline index.html:44]: can we remove this commented code or is going to be used later?

File: _includes/_google-analytics.html (hunk 1 above)
Comment (maciejwalkowiak) [inline _includes/_google-analytics.html:1]: Yep, google analytics will come very soon.

File: index.html (hunk 2 above)
Comment (maciejwalkowiak) [inline index.html:44]: Something to discuss with @florina-muntenescu but I guess we need to tackle pagination quite soon.

**Comment by maciejwalkowiak** [comment]
@florina-muntenescu:
- code is not aligned with a text by design - rather feature than a bug
- new line after picture description added.

**Pull request merged**